packages/ui/src/1_atoms/Tabs/components/Tab/Tab.module.css

```diff
@@ -0,0 +1,68 @@
+.button {
+  @apply font-roboto inline-flex box-border items-center justify-center text-center rounded cursor-pointer transition-colors;
+
+  &.primary {
+    @apply border-t-2 border-t-transparent text-base text-gray-9 rounded-b-none;
+    min-width: 155px;
+    height: 47px;
```

Comment:
We should always use `rem` for styling instead of `px` + `47px` and `155px` seem to me a bit more arbitrary.